pandas/tests/indexes/period/test_ops.py

```diff
@@ -175,9 +175,9 @@ def _check_freq(index, expected_index):
             ordered, indexer = idx.sort_values(return_indexer=True, ascending=False)
             tm.assert_index_equal(ordered, expected[::-1])
 
-            exp = np.array([2, 1, 3, 4, 0])
-            tm.assert_numpy_array_equal(indexer, exp, check_dtype=False)
-            _check_freq(ordered, idx)
+            # GH 35584. Index sort is now stable when sorting in descending order
```

Comment:
can you create a new test and xfail it

Reply:
Added the new test to the end of the file.

Reply:
ok you don't need this comment then

Reply:
Removed the comment.